pkg/scheduler/factory/factory.go

```diff
@@ -892,15 +892,33 @@ func (f *configFactory) CreateFromConfig(policy schedulerapi.Policy) (*scheduler
 	}
 
 	predicateKeys := sets.NewString()
-	for _, predicate := range policy.Predicates {
-		glog.V(2).Infof("Registering predicate: %s", predicate.Name)
-		predicateKeys.Insert(RegisterCustomFitPredicate(predicate))
+	if len(policy.Predicates) == 0 {
```

Comment:
Can you change it to use defaults only when the predicates and priorities are not provided? We would like to keep the possibility of bypassing all predicate or priority functions.

Reply:
Done.